src/validate_pyproject/errors.py

```diff
@@ -9,6 +9,21 @@
 from .error_reporting import ValidationError
 
 
+class URLMissingTool(RuntimeError):
+    _DESC = """\
+    The '--tool' option requires a tool name.
+
+    Correct form is '--tool=<tool-name>={url}', with an optional
```

Comment:
```suggestion
    Correct form is '--tool <tool-name>={url}', with an optional
```

Maybe 1 `=` sign can be a bit easier to understand for users?